core/src/main/java/discord4j/core/spec/MessageCreateSpec.java

```diff
@@ -141,6 +278,16 @@ public MessageCreateSpec setAllowedMentions(@Nullable AllowedMentions allowedMen
         return this;
     }
 
+    /**
+     * Gets the content builder of this spec.
+     * <b>Not recommended! Try to use {@link #getContent} and {@link #setContent} instead!</b>
+     *
+     * @return Content builder of this spec.
+     */
+    public StringBuilder getContentBuilder() {
```

Comment:
Probably not nescessary to expose this. Especially to prevent mutations outside of this class